tools/astarte_import/lib/astarte/import/populatedb/queries.ex

```diff
@@ -253,17 +272,24 @@ defmodule Astarte.Import.PopulateDB.Queries do
     VALUES (?, ?, ?, ?, ?, ?, ?);
     """
 
+    reception_submillis =
+      if(is_nil(reception_timestamp),
+        do: nil,
+        else: rem(DateTime.to_unix(reception_timestamp, :microsecond), 100)
+      )
```

Comment:
nit: extra parentheses

```suggestion
      if is_nil(reception_timestamp),
        do: nil,
        else: rem(DateTime.to_unix(reception_timestamp, :microsecond), 100)
```